voxblox/test/test_approx_hash_array.cc

```diff
@@ -109,6 +109,57 @@ TEST_F(ApproxHashArrayTest, SetRandomWriteRead) {
   EXPECT_GT(recovered, 950);
 }
 
+TEST_F(ApproxHashArrayTest, SetRandomWriteReadLongIndex) {
+  // storing 65536 size_ts
+  ApproxHashSet<16, 10, LongIndex, LongIndexHash> approx_hash_set;
+
+  // generate 1000 random elements
+  AlignedVector<LongIndex> rand_indexes;
+  std::mt19937 gen(1);
+  std::uniform_int_distribution<int64_t> dis(1, 10000000000000000);
```

Comment:
would it make sense to use the max range from [std::numeric_limits](https://en.cppreference.com/w/cpp/types/numeric_limits) ? (same in the old unit test you adapted?)